fawltydeps/settings.py

```diff
@@ -232,15 +232,32 @@ def _option_to_toml(name, value) -> str:  # type: ignore
         Options that are of dictionary type must be given a section entry.
         Assumption: dictionaries options are not nested.
         """
-        comment_char = "# " if has_default_value[name] else ""
+        toml_value_map = {
+            "None": "...",  # always commented, see sanity check below
+            "False": "false",
+            "True": "true",
+            # add more values here for which repr(value) is not valid TOML
+        }
```

Comment:
That's correct. The `toml_value_map` is keyed on the values you get for the settings options, and some of those are not hashable (e.g. lists), so I went the easy way of converting them to strings first. I suppose I can do a try-catch to avoid elaborate checking of types before using `toml_value_map`. What do you think?

Reply:
Opted to go with `repr` instead of try-catching.